app/deed/view/view.mustache

```diff
@@ -0,0 +1,140 @@
+<div class="alert-inline">
+    <p>This is a legal document — please check all details carefully before signing.</p>
+</div>
+
+<article class="deed-document--bounds">
+    <h2 class="heading-large">Mortgage deed</h2>
+
+    <div>
+        <h2 class="heading-medium">Property:</h2>
+
+        <p itemscope="" itemtype="http://schema.org/PostalAddress">
+            <span id="property-address" itemprop="streetAddress">
+                {{land_property.address_1}}<br>
+                {{land_property.address_2}}<br>
+            </span>
+            <span itemprop="addressLocality">{{land_property.city}}</span><br>
+        </p>
+
+        <dl class="definition-inline">
+            <dt>Title number</dt>
+            <dd id="title-number">{{land_property.title_no}}</dd>
+        </dl>
+    </div>
+
+    <div>
+        <div>
+            <h2 class="heading-medium">Lender:</h2>
+        </div>
+
+        <p itemscope="" itemtype="http://schema.org/Corporation">
+            <span itemprop="name">
+                {{lender.name}}
+            </span>
+            <br>
+            <span itemprop="address" itemscope="" itemtype="http://schema.org/PostalAddress" class="address-inline">
+                <span itemprop="streetAddress">
+                    {{lender.address_1}}, {{lender.address_2}},
+                </span>
+                <span itemprop="addressLocality">
+                    {{lender.city}}
+                </span>
+            </span>
+        </p>
+
+        <dl class="definition-inline">
+            <dt>Company registration number</dt>
+            <dd>{{lender.company_no}}</dd>
+        </dl>
+    </div>
+
+    <h2 class="heading-large">Mortgage agreement</h2>
+
+    <div>
+        <h3 class="heading-medium">Borrowers:</h3>
+
+        <div class="grid-row">
+            {{#borrowers}}
+                <div class="column-third">
+                    <dl>
+                        <dd>{{forename}} {{surname}}</dd>
+                        <dd>{{address_1}}</dd>
+                        <dd>{{address_2}}</dd>
+                        <dd>{{city}}</dd>
+                        <dd>{{postcode}}</dd>
+                    </dl>
+                </div>
+            {{/borrowers}}
+        </div>
+    </div>
+
+    <div class="text">
+        <h3 class="heading-small">Charging clause:</h3>
+
+        <p>You, the borrower, with full title guarantee, charge property to the lender by way of legal mortgage with the
+            payment of all money secured by this charge.</p>
+
+        <h3 class="heading-small">Additional provisions:</h3>
+        <ul class="list-bullet">
+            <li>
+                <p>This Mortgage Deed incorporates the <a href="#" rel="external">Lenders Mortgage Conditions and
```

Comment:
If we're linking to something in the app, we should know the URL and link to it - my 2¢